src/plugins/blocksmith/utils/getLines.js

```diff
@@ -25,38 +27,34 @@ const getLinesBrokenByLineBreaks = (txt) => {
  * Calculates the number of lines for a given string using an actual dom node and what happens to
  * it as text is appended.
  *
- * @param {number} spanWidth - the actual width of the text block's dom node
- * @param {string} text      - a string to check for wrapped text
+ * @param {number} node - a node to fill with text to find the wrapped lines
```

Comment:
@efjacobson - change the param type to `Object`

Reply:
@arunkarnati updated to `HTMLElement`